Client/Clique/Common/Services/CreateEventService.swift

```diff
@@ -0,0 +1,75 @@
+import Alamofire
+import UIKit
+
+final class CreateEventService {
+    
+    func getCategories(
+        completion: @escaping(Result<[Category], Error>) -> Void) {
+        AF
+        .request(Constants.Service.categoriesURL, method: .get)
+        .validate(statusCode: 200..<300)
+        .responseData {
+            response in
+            switch response.result {
+            case .success(_):
+                do{
+                    var categories: [Category] = []
+                    let object = response.data
+                    let decoder = JSONDecoder()
+                    let category =  try decoder.decode([Category].self, from: object!)
+                    categories = category
+                    completion(.success(categories))
+                }
+                catch { return }
+            case .failure(let error):
+                completion(.failure(error))
+            }
+        }
+    }
+    
+    func getCurrencies(
+        completion: @escaping(Result<[Currency], Error>) -> Void) {
+        AF
+        .request(Constants.Service.currenciesURL, method: .get)
+        .validate(statusCode: 200..<300)
+        .responseData {
+            response in
+            switch response.result {
+            case .success(_):
+                do{
+                    var currencies: [Currency] = []
+                    let object = response.data
+                    let decoder = JSONDecoder()
+                    let currency =  try decoder.decode([Currency].self, from: object!)
+                    currencies = currency
+                    completion(.success(currencies))
+                }
+                catch { return }
+            case .failure(let error):
+                completion(.failure(error))
+            }
+        }
+    }
+    
+    func createEvent(
+        with entries: CreateEventEntries,
+        completionHandler: @escaping(Bool) -> ()) {
```

Comment:
On other API call, you are using `Result<Something, Error>`, basically, you can do the same here to have the same approach on every call.

In this case, you could define that as:

```suggestion
        completion: @escaping(Result<Void, Error>) -> Void) {
```